pkg/utils/certutil/certutil.go

```diff
@@ -27,6 +27,10 @@ const (
 	Cert_annotation_key   = "cert-manager.io/issuer-name"
 	Bundle_annotation_key = "trust.cert-manager.io/hash"
 	Console_web_prefix    = "webconfig.bindings.artemis."
+
+	// Suffix of provided secrets that contain tls.crt and tls.key items in PEM format.
+	// Secrets with this suffix are not validated. It is useful for secrets created from annotations.
```

Comment:
because they typically won't exist till the annotations are processed and the secret is 'provided' by some other controller.

maybe add in the above text or some variant

Reply:
I added that exact text, thanks for the suggestion